spec/models/invenio_rdm_record_converter_spec.rb

```diff
@@ -164,9 +166,29 @@
           "dates": [{"date": "2021-01-01", "type": {"id":"created"}, "description": "When the item was originally created."}],
           "languages": [{"id": "eng"}, {"id": "pli"}],
           "identifiers": [{
-            "identifier": "10.6666/ARK",
-            "scheme": "ark"
-          }],
+              "identifier": "46589",
+              "scheme": "doi"
+            },
+            {
+              "identifier": "11549777",
+              "scheme": "pmid"
+            },
+            {
+              "identifier": "3956501241",
+              "scheme": "isbn"
+            },
+            {
+              "identifier": "10.6666/ARK",
+              "scheme": "ark"
+            },
+            {
+              "identifier": "PNB-14-96-a",
+              "scheme": "other"
+            },
+            {
+              "identifier": "PNB-15-10",
+              "scheme": "other"
+            }],
```

Comment:
Is `PNB-23-38` missing?

Reply:
Oh, good catch. That test case isn't necessary anymore since it follows the same pattern as "PNB-15-10" of "PNB-integer-integer".